recipes-core/images/iot2050-image-swu-example.bb

```diff
@@ -20,6 +20,7 @@ WKS_FILE = "iot2050-swu.wks.in"
 WKS_FILE:secureboot = "iot2050-swu-secure.wks.in"
 
 IMAGE_FSTYPES += "swu"
+IMAGE_TYPEDEP:swu:append = " wic"
```

Comment:
This should not be required. The cip-core swupdate implementation directly creates the swu file from the squashfs rootfs. If this is needed for the UKI, we should add the dependency in cip-core.

Reply:
It's the UKI, exactly. We could try to include efibootguard.inc from this recipe, let's see...

Reply:
Done.